myst_nbs/generalized_linear_models/GLM-hierarchical-binomial-model.myst.md

```diff
@@ -6,37 +6,34 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python PyMC3 (Dev)
+  display_name: Python 3 (ipykernel)
   language: python
   name: python3
 ---
 
 # Hierarchical Binomial Model: Rat Tumor Example
 :::{post} Nov 11, 2021
-:tags: generalized linear model, hierarchical model, pymc3.Beta, pymc3.Binomial, pymc3.Data, pymc3.Deterministic, pymc3.HalfNormal, pymc3.Model, pymc3.Potential
```

Comment:
@OriolAbril meant to only remove those that started with `pymc.`, not the first 3.